easel/test/widget/publish_screen_test.dart

```diff
@@ -41,5 +62,26 @@ void main() {
       await tester.pump();
       expect(creatorHubViewModel.selectedCollectionType, CollectionType.published);
     });
+
+    testWidgets("Testing refresh published NFT button visibility", (tester) async {
```

Comment:
The test is not related with the ticket. The test should be related to the change of color.

Reply:
I have made changes in the refresh button as well, and I did not find its text so I wrote it.